examples/Three/flipbook3/FlipBook3D.js

```diff
@@ -1,262 +1,297 @@
+    
+            // add page flip interaction TO DO..
+    
+            for (var mii=0; mii<6; ++mii)
             {
-                self.mats[self.sides.front] = new THREE.MeshBasicMaterial({map: self.matFront, overdraw: true});
-                self.mats[self.sides.front].name = "front";
+                // add front - back page images
+                if (mii === self.sides.front)
+                {
+                    self.mats[self.sides.front] = new THREE.MeshBasicMaterial({map: self.matFront, overdraw: true});
+                    self.mats[self.sides.front].map.minFilter = THREE.LinearFilter
+                    self.mats[self.sides.front].name = "front";
+                }
+                else if (mii === self.sides.back)
+                {
+                    self.mats[self.sides.back] = new THREE.MeshBasicMaterial({map: self.matBack, overdraw: true});
+                    self.mats[self.sides.back].map.minFilter = THREE.LinearFilter
+                    self.mats[self.sides.back].name = "back";
+                }
+                else
+                {
+                    self.mats[mii] = new THREE.MeshBasicMaterial({color: self.pageColor});
+                    self.mats[mii].name = 'edge';
+                }
             }
-            else if (mii === self.sides.back)
+            // call super
+            // Three.js has made materials added to FaceMaterial instead of CubeGeometry
+            self.$super("constructor", [new THREE.CubeGeometry(book.pageWidth, book.pageHeight, .5, self.nfacesw, self.nfacesh, .5), new THREE.MeshFaceMaterial(self.mats)]);
+            self.overdraw = true;
+            self.position.x = book.pageWidth * 0.5;
+            self.position.z = -self.zz * self.index;
+    
+            // flip modifiers
+            self.mod = new MOD3.ModifierStack(MOD3.LibraryThree, self);
+            self.mod.addModifier(new MOD3.Pivot(self.position.x, 0, 0)).collapse();
+            self.bend = new MOD3.Bend(0, 0, 0);
+            self.bend.constraint = MOD3.ModConstant.LEFT;
+            self.bend.switchAxes = book.pageHeight > book.pageWidth ? true : false;
+            self.mod.addModifier(self.bend);
+        },
+    
+        book : null,
+        matFront : null,
+        matBack : null,
+        index : null,
+        nfacesw : 10,
+        nfacesh : 10,
+        mats : null,
+        pageHardness : null,
+        angle : null,
+        force : 6,
+        to : null,
+        flipPt : 0.3,
+        mod : null,
+        bend : null,
+        isFlippedLeft : false,
+        isFlippedRight : true,
+        flippingLeft : false,
+        flippingRight : false,
+        zz : 2,
+        pageColor : null,
+        sides : null,
+        onFinishCallback: () => {},
+    
+        flipLeft: function(pt) {
+            var self = this;
+    
+            if (
+                !self.flippingLeft &&
+                !self.flippingRight
```

Comment:
I may need to revise this again. When flipping back and forth rapidly mid-book, things get messed up.